nextcloudappstore/core/forms.py

```diff
@@ -89,3 +94,36 @@ def save(self):
         app_rating.set_current_language(self.cleaned_data['language_code'])
         app_rating.comment = self.cleaned_data['comment']
         app_rating.save()
+
+
+class AppOwnershipTransferForm(ModelForm):
+    to_user = UsernameField(label=_('New owner (username)'))
+    app_id_confirm = CharField(
+        label=_('App ID (for confirmation)'),
+        max_length=App._meta.get_field('id').max_length)
```

Comment:
`_meta` is not private (anymore), but a supported API: https://docs.djangoproject.com/en/1.10/ref/models/meta/